packages/webapp/components/layouts/MainFeedPage.tsx

```diff
@@ -7,14 +7,21 @@ import React, {
 } from 'react';
 import { useRouter } from 'next/router';
 import { MainLayoutProps } from '@dailydotdev/shared/src/components/MainLayout';
-import MainFeedLayout, {
-  MainFeedLayoutProps,
-} from '@dailydotdev/shared/src/components/MainFeedLayout';
+import type { MainFeedLayoutProps } from '@dailydotdev/shared/src/components/MainFeedLayout';
 import AuthContext from '@dailydotdev/shared/src/contexts/AuthContext';
 import { getShouldRedirect } from '@dailydotdev/shared/src/components/utilities';
 import { getFeedName as getFeedNameLib } from '@dailydotdev/shared/src/lib/feed';
+import dynamic from 'next/dynamic';
 import { getLayout } from './FeedLayout';
 
+const MainFeedLayout = dynamic(
+  () =>
+    import(
+      /* webpackChunkName: "goBackHeaderMobile" */ '@dailydotdev/shared/src/components/MainFeedLayout'
+    ),
+  { ssr: false },
```

Comment:
Producing an error about a server-client mismatch. Also, I think no point in doing SSR here since the whole content of this file is purely based on dynamic content.

![Screenshot 2024-07-01 at 8 33 34 PM](https://github.com/dailydotdev/apps/assets/13744167/1a91e8e3-8f9c-43f4-9c08-c7edc417910b)